prime-router/metadata/HL7/catchall/hl7/segments/ORC/OBRExtension.yml

```diff
@@ -192,6 +192,18 @@ extension:
           type: STRING
           expressionType: HL7Spec
           valueOf: OBR.24
+    - expressionType: nested
+      vars:
+        obr25: OBR.25
+      condition: $obr25 NOT_NULL
+      expressionsMap:
+        url:
+          type: STRING
+          value: OBR.25
+        valueId:
+          type: STRING
+          expressionType: HL7Spec
+          valueOf: OBR.25
```

Comment:
Added this in, but didn't need to update any tests as this was already getting mapped through DiagnosticReport. I'm guessing we are fine with multiple mappings of FHIR->HL7 for the same field through different fhir resources as they should all line up in the end. https://github.com/CDCgov/prime-reportstream/pull/15333/commits/d34d8a34286273d420115e2793b83dece583d699

Reply:
Huddled with Brick. The reason the tests were passing was because of the ordering of the fhir->hl7 mappings. Changing the ordering creates the same test failure effect as xad.6 did elsewhere. The OBRExtension mapping _is_ required because the hl7->fhir mappings is from many to one. Many HL7 values map to the fhir value for DiagnosticReport.status of 'unknown' and thus the hl7 would be lost without adding in this extension. 

Todo: For DR.Status mapping of FHIR->HL7 we need to add a condition that it will not be mapped if the value is unknown so that it can pull from the ServiceRequest extension where OBR.25 is preserved.

Reply:
After bringing this up in standup last week the correct approach was decided to use OBRExtension as the source of truth for OBR.25. This means OBRExtension needs to be added to FHIR sender transforms. 

https://github.com/CDCgov/prime-reportstream/pull/15333/commits/788c0e2b1617c660ea4bf43140ff1defdbbe51e4